src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs

```diff
@@ -279,4 +297,14 @@ private void UpdateRandomButtonVisibility()
             randomizeNameButton.Visible = showRandomizeButton;
         }
     }
+
+    private void OnControlMouseEntered()
```

Comment:
I kinda don't like this solution. Isn't there some other way to make sure the validation is for example performed *again* when clicking the random button or something to fix the problem?

Reply:
I don't like the solution, but as far as I can tell, there is no other good option. I know "validation is for example performed again when clicking the random button" has a couple of issues that would stop it from working.

Reply:
Care to elaborate? I think it anyway would make sense that the random button would retrigger the validation.
Oh and now that I'm thinking, shouldn't the Clicked callback ensure it doesn't trigger the validation unnecessarily a bunch of times? It should only happen *once* after the text has been changed in the name box.